metrics.go

```diff
@@ -150,13 +152,31 @@ func main() {
 	handler.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 	handler.HandleFunc("/debug/pprof/trace", pprof.Trace)
 
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
+	go func() {
+		for sig := range c {
+			// sig is a ^C, handle it
+			log.Printf("caught sig: %+v", sig)
+			log.Println("Closing http server and listener")
```

Comment:
You should have the same logs in both applications. This one has an extra line that isn't present in the event consumer.

Reply:
Removed in log.Println("Closing http server and listener") from metrics